app/[...slug]/page.tsx

```diff
@@ -14,7 +14,7 @@ type ParamsType = {
   slug: string[];
 };
 
-async function generateStaticParams() {
+export async function generateStaticParams() {
```

Comment:
To use static export, I need to add `export` back here. Previously it generates a type error, but it seems ok now.